tests/acceptance/features/webUIPreview/imageMediaViewer.feature

```diff
@@ -14,3 +14,32 @@ Feature: display image in media viewer on the webUI
     And user "user1" has logged in using the webUI
     When the user views the file "test_video.mp4" in the mediaviewer using the webUI
     Then the file "test_video.mp4" should be displayed in the mediaViewer
+
+  Scenario Outline: navigate to next and previous videos with media viewer is possible
+    Given user "user1" has uploaded file "<resource-one>" to "<resource-one-name>"
+    And user "user1" has uploaded file "<resource-two>" to "<resource-two-name>"
+    And user "user1" has logged in using the webUI
+    When the user views the file "<resource-one-name>" in the mediaviewer using the webUI
+    And the user navigates to the next media resource using the webUI
+    Then the file "<resource-two-name>" should be displayed in the mediaViewer
+    When the user navigates to the previous media resource using the webUI
+    Then the file "<resource-one-name>" should be displayed in the mediaViewer
+    Examples:
+      | resource-one   | resource-one-name | resource-two    | resource-two-name |
+      | test_video.mp4 | test_video0.mp4   | test_video1.mp4 | test_video1.mp4   |
+      | testavatar.jpg | testavatar.jpg    | testavatar.png  | testavatar.png    |
```

Comment:
I think you can reuse `resource-one` and `resource-two`. And you will not need `resource-one-name` and `resource-two-name` :thinking: :thought_balloon: